lc-hax/Scripts/Modules/Possession/Controllers/BaboonHawkController.cs

```diff
@@ -8,6 +9,30 @@ enum BaboonState {
 }
 
 internal class BaboonHawkController : IEnemyController<BaboonBirdAI> {
+
+    Vector3 customCamp = new(1000, 1000, 1000);
+
+    Vector3 originalCamp = new(0, 0, 0);
+
+    public void OnDeath(BaboonBirdAI enemyInstance) {
+        if (enemyInstance.heldScrap is not null) {
+            _ = enemyInstance.Reflect().InvokeInternalMethod("DropHeldItemAndSync");
+        }
+    }
+
+    public void OnPossess(BaboonBirdAI enemyInstance) {
+        if (BaboonBirdAI.baboonCampPosition != this.customCamp) return;
+        this.originalCamp = BaboonBirdAI.baboonCampPosition;
```

Comment:
Why do we have to move the baboon's camp?

Reply:
To make it not drop stuff when we hold it